examples/src/main/scala/org/apache/spark/examples/mllib/DenseGaussianMixture.scala

```diff
@@ -18,17 +18,17 @@
 package org.apache.spark.examples.mllib
 
 import org.apache.spark.{SparkConf, SparkContext}
-import org.apache.spark.mllib.clustering.GaussianMixtureEM
+import org.apache.spark.mllib.clustering.GaussianMixture
 import org.apache.spark.mllib.linalg.Vectors
 
 /**
  * An example Gaussian Mixture Model EM app. Run with
  * {{{
- * ./bin/run-example org.apache.spark.examples.mllib.DenseGmmEM <input> <k> <covergenceTol>
+ * ./bin/run-example org.apache.spark.examples.mllib.DenseGaussianMixture <input> <k> <convergeTol>
```

Comment:
`convergenceTol` (`run-example` doesn't need `org.apache.spark.examples.`, so you can still make it in one line.)

~~~
./bin/run-examples mllib.DenseGaussianMixure <input> <k> <convergenceTol>
~~~